knowledge_repo/converters/html.py

```diff
@@ -101,8 +100,8 @@ def run(self, parent, blocks):
 
 class IndentsAsCellOutput(Extension):
 
-    def extendMarkdown(self, md, md_globals):
-        md.preprocessors.add("code_isolation", IndentsAsCellOutputPreprocessor(md), "<html_block")
+    def extendMarkdown(self, md):
```

Comment:
Same as above; if we use the old code, we are less sensitive to upstream changes and can avoid magic numbers.